src/content/json/header-assets.js

```diff
@@ -0,0 +1,49 @@
+import blueprintIcon from "../images/blueprint_banner_negative.webp";
+import styled from 'styled-components';
+
+export const BlueprintIcon = () => {
+  return (
+    <img style={{ maxWidth: '150px' }} src={blueprintIcon}></img>
+  );
+};
+
+export const StyledHeader = styled.div`
+  background-color: #3988ff;
```

Comment:
Just noting that this should eventually be re-factored to use the theme provider when that is implemented in the future.